challenges/lib/src/main/java/challenges/stackandqueues/PseudoQueue.java

```diff
@@ -0,0 +1,25 @@
+package challenges.stackandqueues;
+
+public class PseudoQueue {
+public Stack front;
+public Stack back;
+
+    public PseudoQueue() {
+        this.front = new Stack();
+        this.back = new Stack();
+    }
+    public void enqueue (int input){
+
+        back.push(input);
+    }
+    public int dequeue () throws Exception {
+        if (this.front.isEmpty() == true){

```

Comment:
within the if statement, you don't need to set isEmpty() to == true. The method already returns a boolean. If you want to check for falsey, just do !isEmpty() (is not empty)